esp/src/eclwatch/Utility.js

```diff
@@ -19,6 +20,13 @@
         ;
     }
 
+    function csvEncode(str) {
+        str = "" + str;
+        return str.replace(/"/g, '\"')
+                  .replace(/','/g, '\'')
+        ;
+    }
```

Comment:
I would imagine it should look like this:
```
function csvEncode(cell) {
    if (!isNaN(cell)) return cell;
    return '"' + String(cell).replace('"', '""') + '"';
}
```
I would also suggest moving it to Utility (where we already have xmlEncode) and adding a bunch of tests like we do here:  https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/eclwatch/Utility.js#L122-L137